ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -190,26 +205,28 @@ public struct SignalProducer<T, E: ErrorType> {
 		// Create a composite disposable that will automatically be torn
```

Comment:
This is another case where I think calling the above observer either sink or pipe would improve clarity in this function.